src/app/views/collections/validation/collectionValidation.js

```diff
@@ -33,6 +33,13 @@ export default class collectionValidation {
             };
         }
 
+        if (collections && collections.some(c => c.id.split("-")[0] === name.replace(/\s+/g, ""))) {
+            response = {
+                isValid: false,
+                errorMsg: "A collection with this ID already exists. Please choose a different name.",
```

Comment:
Not sure if we should add this error to `errCodes` like for unique name?

Reply:
> This new validation is only needed when a collection has been renamed. Only in that case the name and id won't match. However, if the collection has not been renamed, I think the existing unique name error will stop showing and this will show instead as it overrides the `response` object. Is this the expected behaviour? Could we add unit test for that scenario?

Hello, 

Just so I understand the scenario you are suggesting because it took me a few passes to grasp it lol

If a collection is called alpha thus id: alpha and name: alpha. 

Then a second collection tries to be created called alpha.

Is the problem that the UNIQ_NAME_ERROR will never show and it will default to the "A collection with this ID already exists. Please choose a different name." error message? 

If that is your thinking then to be honest I don't know what the desired behaviour is in that instance and will need to check with the PO.

Reply:
Yes that is the scenario I was thinking about. Please check with the PO and if we are happy with that result, I think the existing name check is dead code and should be removed

Reply:
We should also add an unit test for that scenario. I notice the examples in the tests never have an id and name matching (and this is what most collection will have)

Reply:
Would it be dead code?

I mean what about a scenario where

A collection called alpha gets created thus id: alpha-123 and name: alpha

Then a second collection gets created called alphaTest thus id: alphaTest-123 and name: alphaTest

Then alphaTest gets renamed to alpha. Removing the existing name check would allow for that and cause issues, no?

Reply:
Yes, I think you are right there! 👍